settings.py

```diff
@@ -569,28 +569,11 @@
 }
 
 CELERY_MAIN_QUEUE = 'celery'
-
-# this is the default celery queue
-# for periodic tasks on a separate queue override this to something else
-CELERY_PERIODIC_QUEUE = CELERY_MAIN_QUEUE
-
-# This is the celery queue to use for running reminder rules.
-# It's set to the main queue here and can be overridden to put it
-# on its own queue.
-CELERY_REMINDER_RULE_QUEUE = CELERY_MAIN_QUEUE
-
-# This is the celery queue to use for running reminder case updates.
-# It's set to the main queue here and can be overridden to put it
-# on its own queue.
-CELERY_REMINDER_CASE_UPDATE_QUEUE = CELERY_MAIN_QUEUE
-
-
-# This is the celery queue to use for sending repeat records.
-# It's set to the main queue here and can be overridden to put it
-# on its own queue.
-CELERY_REPEAT_RECORD_QUEUE = CELERY_MAIN_QUEUE
```

Comment:
Flagging that it looks like this one isn't overridden in pna, swiss, or l10k.

Reply:
That's right. These lines
- https://github.com/dimagi/commcare-cloud/blob/ef6c0eaa4bcbc9926013120e3d30c656dfb47614/environments/l10k/app-processes.yml#L10
- https://github.com/dimagi/commcare-cloud/blob/ef6c0eaa4bcbc9926013120e3d30c656dfb47614/environments/pna/app-processes.yml#L10
- https://github.com/dimagi/commcare-cloud/blob/ef6c0eaa4bcbc9926013120e3d30c656dfb47614/environments/swiss/app-processes.yml#L10
say "run the `repeat_record_queue` queue in the same process as the `celery` queue", which is what not overriding `CELERY_REPEAT_RECORD_QUEUE` used to do.

That sound right?